models/comments.model.js

```diff
@@ -15,3 +15,18 @@ exports.fetchCommentsByArticleId = (article_id) => {
       return result.rows;
     });
 };
+
+exports.insertCommentByArticleId = (article_id, username, body) => {
+  return db
+    .query(
+      `
+          INSERT INTO comments (author, article_id, body, created_at)
+          VALUES ($1, $2, $3, NOW())
```

Comment:
I think the date gets generated by psql automatically, so optionally could remove the NOW - no harm in having it though, but just something to be aware of